astropy/io/misc/ecsv.py

```diff
@@ -74,6 +75,8 @@
 
 __all__ = ["read_ecsv", "register_ecsv_table", "write_ecsv"]
 
+ECSVEngines: dict[str, "ECSVEngine"] = {}
```

Comment:
🧐:  feel free to close. Just signifies this can be mutated, but should not be re-assigned.

```suggestion
ECSVEngines: Final[dict[str, "ECSVEngine"]] = {}
```